src/index.js

```diff
@@ -441,6 +479,24 @@ class EventPortal {
     }
   }
 
+  /**
+  * Return the schema version object given the schemaVersionId
+  * In this case we don't have the parent schema ID, so we need to get it from the schemaVersionId
+  * * Warning * : This is not an offically supported Event Portal API
```

Comment:
Same as above, I had to use a "hidden" API to get the Schema Version Object. I don't have the parent schema id so I had to use the "schemaVersions" API to get the schema version object directly using the schemaVersionId.